Backend/autozon/app/Http/Controllers/CarController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\DB;
+
+class CarController extends Controller
+{
+    /**
+   * Create a new controller instance.
+   * @return void
+   */
+  public function __construct() {}
+
+  /**
+   * Show all car models
+   * @return carModel[]
+   */
+  public function getAllCarBrands() {
```

Comment:
Check to only get each brand once. E.g. "VW" can be 10 times in this table. Only return once